README.md

```diff
@@ -1,8 +1,200 @@
-### YAGNA DAPP MANAGER
+# Golem dApp Manager
 
-Manage a dapp.
+`dapp-manager` is a purposefully minimalistic manager for decentralized applications running on
+Golem. It works together with the [dapp-runner](https://github.com/golemfactory/dapp-runner/).
+While the responsibility of the latter is to run a single Golem application, `dapp-manager` takes
+care of spawning, interacting with, and stopping the running instances of the `dapp-runner`.
+
+## Quick start
+
+### Yagna daemon
+
+As the `dapp-manager` uses the `dapp-runner`, which in turn requires a properly configured
+[yagna](https://github.com/golemfactory/yagna) daemon, you'll need to have it set up.
```

Comment:
"you'll seen to have it set up" feels out of place in this sentence

Reply:
@nzmel what do you mean, it's "need", not "seen" and it's a requirement to have the yagna daemon set-up before you're able to run `dapp-manager` ... what's out of place?